lib/matplotlib/type1font.py

```diff
@@ -264,19 +263,19 @@ def fontmatrix(array):
             newmatrix = np.dot(modifier, oldmatrix)
             array[::2] = newmatrix[0:3, 0]
             array[1::2] = newmatrix[0:3, 1]
```

Comment:
Not sure the byte literal approach is an improvement here. Gets a bit cluttered.